config/jest/jest.config.js

```diff
@@ -25,5 +25,10 @@ module.exports = {
     '<rootDir>/config/jest/polyfill.rAF.js',
     '<rootDir>/config/jest/enzyme.config.js'
   ],
-  testEnvironment: '<rootDir>/config/jest/jsdom.config.js'
+  testEnvironment: '<rootDir>/config/jest/jsdom.config.js',
+  transform: {
```

Comment:
1. Why do you need the `transform` config?
2. What does the `babel-jest` part do?
3. What does the `assetsTransformer.js` part do?

Reply:
The transform config is needed in conjunction with the assetsTransformer.js in order to essentially mock any imported assets like SVGs, images, etc as they are not particularly useful for testing. In this particular case, it was to handle the case where we import and display the OpenLattice logo in AppContainer.js

https://github.com/facebook/jest/issues/2663#issuecomment-317109798